vdr/doc-creator.go

```diff
@@ -0,0 +1,111 @@
+package vdr
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/lestrrat-go/jwx/jwk"
+	"github.com/shengdoushi/base58"
+
+	"github.com/nuts-foundation/go-did"
+
+	nutsCrypto "github.com/nuts-foundation/nuts-node/crypto"
+)
+
+const NutsDIDMethodName = "nuts"
+
+// NutsDocCreator implements the DocCreator interface and can create Nuts DID Documents.
+type NutsDocCreator struct {
```

Comment:
Remove the `nuts` prefix

Reply:
Is is actually the NutsDockCreator implementation of a DocCreator interface. It uses the `nuts` method string and implements the DID method according to the spec. So I don't think it is redundant.